2018-05-04 15:07:10  [ main:0 ] - [ ERROR ]  {dataSource-1} init error
java.sql.SQLException: com.mysql.jdbc.Driver
	at com.alibaba.druid.util.JdbcUtils.createDriver(JdbcUtils.java:600)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:815)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1227)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1223)
	at DbUtilsDruid.init(DbUtilsDruid.java:47)
	at DbUtilsDruid.<init>(DbUtilsDruid.java:39)
	at DbUtilsDruid.<init>(DbUtilsDruid.java:23)
	at DbUtilsDruid$SingletonHolder.<clinit>(DbUtilsDruid.java:30)
	at DbUtilsDruid.getInstance(DbUtilsDruid.java:35)
	at DbUtilsDruid.main(DbUtilsDruid.java:170)
Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:338)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at com.alibaba.druid.util.JdbcUtils.createDriver(JdbcUtils.java:598)
	... 9 more
2018-05-04 15:07:10  [ main:26 ] - [ ERROR ]  没有查到任何结果，有问题
2018-05-04 15:08:26  [ main:0 ] - [ ERROR ]  init datasource error, url: jdbc:mysql://localhost:3306/demolll
java.sql.SQLSyntaxErrorException: Unknown database 'demolll'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:536)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:115)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:1606)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:633)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1515)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1578)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:859)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1227)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1223)
	at DbUtilsDruid.init(DbUtilsDruid.java:47)
	at DbUtilsDruid.<init>(DbUtilsDruid.java:39)
	at DbUtilsDruid.<init>(DbUtilsDruid.java:23)
	at DbUtilsDruid$SingletonHolder.<clinit>(DbUtilsDruid.java:30)
	at DbUtilsDruid.getInstance(DbUtilsDruid.java:35)
	at DbUtilsDruid.main(DbUtilsDruid.java:170)
2018-05-04 15:08:26  [ main:13 ] - [ ERROR ]  {dataSource-1} init error
java.sql.SQLSyntaxErrorException: Unknown database 'demolll'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:536)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:115)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:1606)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:633)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1515)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1578)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:859)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1227)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1223)
	at DbUtilsDruid.init(DbUtilsDruid.java:47)
	at DbUtilsDruid.<init>(DbUtilsDruid.java:39)
	at DbUtilsDruid.<init>(DbUtilsDruid.java:23)
	at DbUtilsDruid$SingletonHolder.<clinit>(DbUtilsDruid.java:30)
	at DbUtilsDruid.getInstance(DbUtilsDruid.java:35)
	at DbUtilsDruid.main(DbUtilsDruid.java:170)
2018-05-04 15:08:26  [ main:17 ] - [ INFO ]  {dataSource-1} inited
2018-05-04 15:08:26  [ Druid-ConnectionPool-Create-1414521932:20 ] - [ ERROR ]  create connection SQLException, url: jdbc:mysql://localhost:3306/demolll, errorCode 1049, state 42000
java.sql.SQLSyntaxErrorException: Unknown database 'demolll'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:536)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:115)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:1606)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:633)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1515)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1578)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2466)
2018-05-04 15:08:26  [ Druid-ConnectionPool-Create-1414521932:23 ] - [ ERROR ]  create connection SQLException, url: jdbc:mysql://localhost:3306/demolll, errorCode 1049, state 42000
java.sql.SQLSyntaxErrorException: Unknown database 'demolll'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:536)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:115)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:1606)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:633)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:149)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:143)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1515)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1578)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2466)
2018-05-04 15:08:26  [ main:30 ] - [ ERROR ]  没有查到任何结果，有问题
2018-05-04 15:08:40  [ main:0 ] - [ INFO ]  {dataSource-1} inited
2018-05-04 15:08:40  [ main:84 ] - [ INFO ]  0
2018-05-04 15:08:59  [ main:2 ] - [ INFO ]  {dataSource-1} inited
2018-05-04 15:08:59  [ main:80 ] - [ INFO ]  0
